mesheryctl/cmd/perf.go

```diff
@@ -82,6 +85,58 @@ func StringWithCharset(length int) string {
 	return string(b)
 }
 
+func AddAuthDetails(req *http.Request, filepath string) error {
+	file, err := ioutil.ReadFile(filepath)
+	if err != nil {
+		log.Errorf("File read failed : %v", err.Error())
+		return err
+	}
+	var tokenObj map[string]string
+	if err := json.Unmarshal(file, &tokenObj); err != nil {
+		log.Errorf("Token file invalid : %v", err.Error())
+		return err
+	}
+	req.AddCookie(&http.Cookie{
+		Name:     tokenName,
+		Value:    tokenObj[tokenName],
+		HttpOnly: true,
+	})
+	req.AddCookie(&http.Cookie{
+		Name:     providerName,
+		Value:    tokenObj[providerName],
+		HttpOnly: true,
+	})
+	return nil
+}
+
+func UpdateAuthDetails(filepath string) error {
+	req, err := http.NewRequest("GET", mesheryAuthToken, bytes.NewBuffer([]byte("")))
+	if err != nil {
+		return err
+	}
+	if err := AddAuthDetails(req, filepath); err != nil {
+		return err
+	}
+
+	client := &http.Client{}
+	resp, err := client.Do(req)
+	defer SafeClose(resp.Body)
+
+	if err != nil {
+		return err
+	}
+
+	data, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return err
+	}
+
+	if err := ioutil.WriteFile(filepath, data, os.ModePerm); err != nil {
+		return err
```

Comment:
No need to check whether err!= nil,
Please return error itself in the next line